src/cipher.jl

```diff
@@ -88,15 +88,15 @@ function add_padding_PKCS5(data::Vector{UInt8}, block_size::Int)
 end
 
 function trim_padding_PKCS5(data::Vector{UInt8})
-  padlen = data[sizeof(data)]
+    padlen = data[end]
   if all(data[end-padlen+1:end-1] .== data[end])
-    return data[1:sizeof(data)-padlen]
+        return data[1:end-padlen]
```

Comment:
I think you need to fix the indentation in the entire function here.

Reply:
Fixed!